### PR TITLE
locale: add page

### PR DESCRIPTION
**Pull request by Managor**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/locale.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# locale
+
+> Get locale-specific information.
+> More information: <https://man7.org/linux/man-pages/man1/locale.1.html>.
+
+- List the global locale variables and their values:
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/locale.md:6]: Yes

[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/linux/locale.md:6]: Based on the man page, I think the example description should be something similar to this.
```suggestion
- List values for environment variables for each locale category setting:
```
But, I don't like this suggestion at all. Let's wait for another reviewer.

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/linux/locale.md:6]: How about
```suggestion
- List all global environment variables, describing the user's locale:
```

[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/locale.md:6]: Is the comma grammatically correct?

[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/linux/locale.md:6]: Looks like you can write it with and without. I've edited my suggestion above to remove it.

[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/locale.md]: The one with the comma sounded like the act of listing is describing the locale while without it sounds more like the listed variables themselves are doing it.

**Review by marchersimon — APPROVED** [review]
Thanks @Managor

**Review by CleanMachine1 — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**